test/test-pipe-win-uds.c

```diff
@@ -0,0 +1,183 @@
+/* Copyright Joyent, Inc. and other Node contributors. All rights reserved.
```

Comment:
That's the old copyright. For new tests, you can use this:
```suggestion
/* Copyright libuv contributors. All rights reserved.
```
(Joyent hasn't been a thing here for 12 or 13 years now.)